packages/oscal-react-library/src/components/OSCALAnchorLinkHeader.tsx

```diff
@@ -12,7 +12,16 @@ const AnchorLinkIcon = styled(LinkIcon)(({ theme }) => ({
   color: theme.palette.grey[500],
 }));
 
-export default function OSCALAnchorLinkHeader(props) {
+export interface AnchorLinkProps {
+  urlFragment?: string;
+}
+
+export interface OSCALAnchorLinkHeaderProps {
```

Comment:
Is there any reason for this to be exported? I think this interface is just used in this file?

Reply:
If a component is exported, its interface should also be exported.